src/definitions/modules/toast.js

```diff
@@ -389,13 +389,15 @@ $.fn.toast = function(parameters) {
                   onBeforeHide: function(callback){
                       callback = $.isFunction(callback)?callback : function(){};
                       if(settings.transition.closeEasing !== ''){
-                          $toastBox.css('opacity',0);
-                          $toastBox.wrap('<div/>').parent().slideUp(500,settings.transition.closeEasing,function(){
-                            if($toastBox){
-                              $toastBox.parent().remove();
-                              callback.call($toastBox);
-                            }
-                          });
+                          if($toastBox) {
+                            $toastBox.css('opacity', 0);
+                            $toastBox.wrap('<div/>').parent().slideUp(500, settings.transition.closeEasing, function () {
+                              if ($toastBox) {
```

Comment:
To avoid the error, 1st `if` seems enough for me.
Why 2nd one is needed?

Reply:
That one was alreadyl there and will also happen 500ms after the slideUp. So in some circumstances, the access to $toastBox could fail here as well (thats why i put it there before already). Lets keep it there

Reply:
@exoego are you approving as well now? 😉